### DISCUSSION TITLE
blog/what-is-developer-experience/

### DISCUSSION THREAD

**Discussion by giscus** [verifa/website#110] [Blog]
# blog/what-is-developer-experience/

Developer Experience promises improved productivity, satisfaction, engagement and retention of development teams. We conducted a study of existing material and in this post we share the summary together with our real-world experience.

https://verifa.io/blog/what-is-developer-experience/

**Comment by lakruzz** [comment]
Well done @praqma-thi and @jlarfors 💪

I will take the liberty and use your article as a scholar read in my team at @thetechcollective. ...and brag about the fact that I know the authors personally!

I'm so proud of you both 👴 🤣

**Reply by jlarfors** [reply]
@lakruzz thanks for your kind words 🙏 and happy that the article is useful to you at @thetechcollective - that's awesome! Looks like you're doing great things there.
